Cargo.toml

```diff
@@ -58,6 +58,9 @@ nus3audio = "1.2.0"
 # For motion list patching
 motion_list_rs = "1.4.3"
 serde_yaml = "0.8"
+# for bgm property patching
+smash-bgm-property = "1.0.1"
+binread = "1.4.1"
```

Comment:
Considering you have a crate specifically to deal with bgm_property, why introduce the dependency just for a trait here?
Offer a ::open method instead on the struct in the crate instead